cl/users/templates/emails/neon_multiple_accounts.txt

```diff
@@ -0,0 +1,20 @@
+A new user signed up for CourtListener and we tried to match their email address
+to Neon. Alas, it didn't work because the email address {{ user.email }} appears
+is in Neon more than once.
+
+Please take the following actions to fix this situation:
+
+i. Please review the "Partial Match Queue" in Neon, as documented here:
```

Comment:
```suggestion
A new user signed up for CourtListener and we tried to match their email address
to Neon. Alas, it didn't work because the email address '{{ user.email }}' appears
is in Neon more than once.

Please take the following actions to fix this situation:

1. Please review the "Partial Match Queue" in Neon, as documented here:
```